apps/web/src/features/counterfactual/hooks/usePendingSafeStatuses.ts

```diff
@@ -57,7 +61,7 @@ const usePendingSafeMonitor = (): void => {
           } = undeployedSafe
 
           const isProcessing = status === PendingSafeStatus.PROCESSING && txHash !== undefined
-          const isRelaying = status === PendingSafeStatus.RELAYING && taskId !== undefined
+          const isRelaying = status === PendingSafeStatus.RELAYING
```

Comment:
We could consider adding a new `PendingSafeStatus.SAFENET_RELAYING` in case we run into problems with re-using `RELAYING`